src/java.base/share/classes/javax/security/auth/Subject.java

```diff
@@ -855,7 +851,7 @@ public <T> Set<T> getPublicCredentials(Class<T> c) {
      * {@code Class}.
      *
      * <p> If a security manager is installed, the caller must have a
-     * {@link PrivateCredentialPermission} to access all of the requested
+     * {@link PrivateCredentialPermission} to access all the requested
```

Comment:
Hmm. I think the previous text was more correct/readable.

Reply:
Okay. I'll revert the change. It was just an IJ suggestion.